src/server/components/lists/helpers.ts

```diff
@@ -268,17 +242,32 @@ export function formatCountryParam(country: string): string {
   let countryName: string = country;
 
   if (countryName) {
-    countryName = startCase(country)
+    countryName = startCase(country);
     if (countryName === "Northern Cyprus") {
       countryName = "northern Cyprus";
     }
     if (countryName === "Cote D Ivoire") {
       countryName = "Côte d'Ivoire";
     }
     const specialChars = [".", ",", "-", "ã", "é", "í", "ç", "ô"];
-    specialChars.forEach(specialChar => {
+    specialChars.forEach((specialChar) => {
       countryName = restoreSpecialCharacter(specialChar, country, countryName);
     });
   }
   return countryName;
 }
+
+function getRestOfQueryParams(req: Request): string {
+  const { serviceType, country, ...rest } = req.query;
+  return querystring.stringify(rest as NodeJS.Dict<string>);
+}
+
+export function redirectToNewUrlStructure(req: Request, res: Response, next: NextFunction) {
+  const oldUrlStructure = "/results?serviceType=";
```

Comment:
it assumes here that serviceType would be the first query parameter.  By and large that should be the case but if someone was to manually hack the URL the redirect wouldn't take place.  I'd suggest you add another check for the country param as well.  However, see my other comment on middleware and implement in the router.